test/utility_tests.jl

```diff
@@ -21,4 +21,10 @@ using WaspNet
 
     include("pruning_tests.jl")
 
+    @test begin
+        inputs = [1,1]
```

Comment:
Try testing the case where the input vector is deterministic as well, like an `inputs` vector of `[0, 0, 0, 1]`.